### DISCUSSION TITLE
Add support for MVTEC LOCO AD

### DISCUSSION THREAD

**Discussion by ORippler** [open-edge-platform/anomalib#574] [Feature Requests]
**Is your feature request related to a problem? Please describe.**

MVTec AD dataset only consists of low-level, textural/structural anomalies. To test the capability at detecting high-level, semantic/logical anomalies, additional datasets are needed.

**Describe the solution you'd like**

Integrate a default dataloader and evaluation scheme for [MVTEC LOCO AD](https://link.springer.com/article/10.1007/s11263-022-01578-9#Fn1) dataset

**Comment by willyfh** [comment]
Hello @samet-akcay, is there any plan to implement MVTEC LOCO AD? I would like to work on this if it's okay. As far as I know, this pull request (PR) has been discontinued: https://github.com/openvinotoolkit/anomalib/pull/553

**Comment by samet-akcay** [comment]
Hi @willyfh, thanks for your interest! There is currently no one working on this. @jpcbertoldo has some other priorities
